agent.go

```diff
@@ -665,5 +720,7 @@ func main() {
 	// Start gRPC server.
 	s.startGRPC()
 
+	go s.listenToUdevEvents()
```

Comment:
please let s.wg.Wait() also wait for the udev listener goroutine.

Reply:
The udev listener routine does not quit for any reason, thats why I have not added the wait for it.
It should quit when the main thread quits.